README.md

```diff
@@ -1,24 +1,199 @@
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# 📖 Cash In-Out App <a name="about-project"></a>
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+"**Cash In-Out** is a Ruby on Rails financial management application designed to help users efficiently track their income and expenses. The application enables users to create and manage transactions, categorize them, and visualize their financial activities. With features like user authentication, transaction editing, and category organization, Cash In-Out empowers users to maintain a clear overview of their financial transactions, facilitating effective budgeting and financial planning."
 
-* Deployment instructions
+## Live Demo & Presentation <a name="live-demo">
+[LIVE DEMO](https://cash-in-out-qoka.onrender.com/)
 
-* ...
+<!-- Video Presentation -->
+## Video Presentation <a name="video-presentation"></a>
+[See Video](https://vimeo.com/883597169/4195c3d8ef?share=copy)
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://guides.rubyonrails.org/">Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details> 
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add categories**
+- **Add transactions**
+- **Mobile friendly UI**
+- **Delete Account**
+- **Topup credit**
+- **Clear credit**
+- **Balance display**
+- **User Authentication**
+- **Splash screen**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+- Code editor like VS code
+- Ruby installed in your machine
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+   git clone https://github.com/geekelo/cash-in-out.git
+   cd cash-in-out
+```
+
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd cash-in-out
+  gem install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```sh
+  rails server
+```
+
+### Run Tests
+To run tests, run the following command:
+```
+  bin/rails test test/models/article_test.rb
+```
```

Comment:
- [ ] It would be great to include ```rspec spec``` to run the specs which is the command to use since you are using rspec for your unit tests.